internal/service/advancedcluster/data_source_schema.go

```diff
@@ -0,0 +1,731 @@
+package advancedcluster
+
+import (
+	"context"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+)
+
+func DataSourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"accept_data_risks_and_force_replica_set_reconfig": schema.StringAttribute{
+				Computed:            true,
+				Description:         "If reconfiguration is necessary to regain a primary due to a regional outage, submit this field alongside your topology reconfiguration to request a new regional outage resistant topology. Forced reconfigurations during an outage of the majority of electable nodes carry a risk of data loss if replicated writes (even majority committed writes) have not been replicated to the new primary node. MongoDB Atlas docs contain more information. To proceed with an operation which carries that risk, set **acceptDataRisksAndForceReplicaSetReconfig** to the current date.",
```

Comment:
do you plan to incorporate description handling changes from https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2605 at a later point?
Is that something that can be accommodated in the scaffolding as well?

Reply:
Correct, not aware of any customization we can do with the current tooling for this